ca/ca.go

```diff
@@ -492,8 +493,24 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(ctx context.Context, csr x5
 		return emptyCert, err
 	}
 
+	var ocspResp []byte
+	if features.Enabled(features.GenerateOCSPEarly) {
+		ocspResp, err = ca.GenerateOCSP(ctx, core.OCSPSigningRequest{
+			CertDER: certDER,
+			Status:  "good",
+		})
+		if err != nil {
+			err = berrors.InternalServerError(err.Error())
+			ca.log.AuditInfo(fmt.Sprintf("OCSP Signing failure: serial=[%s] pem=[%s] err=[%s]",
+				serialHex, certPEM, err))
+			// If OCSP generation fails don't return an error and orphan the
```

Comment:
will this really orphan the certificate in the sense we use the word elsewhere? I might rephrase otherwise to avoid overloading with the concept that the cert won't make it through to the DB.